app/src/androidTest/java/com/example/android/codelabs/agera/step2/Step2ActivityFinalTest.java

```diff
@@ -44,22 +44,22 @@
 
     @Test
     public void incrementButtonPressed_textIsUpdated() {
-        onView(withId(R.id.step2_value_tv)).check(matches(withText("0")));
-        onView(withId(R.id.increment_bt)).perform(click());
-        onView(withId(R.id.step2_value_tv)).check(matches(withText("1")));
+        onView(withId(R.id.step2_value)).check(matches(withText("0")));
```

Comment:
There is no need to use type(such as `tv`, `bt`) in IDs, and in the entire source code of Android SDK is not seen so named. :P